tools/commands/build.py

```diff
@@ -127,6 +132,33 @@ def make(self):
         logger.info("building & installing {}: {}".format(self.name, " ".join(cmd)))
         subprocess.check_call(cmd, env=self.env)
 
+# hostap builder class
+class hostapbuilder(builder):
+    def __init__(self, name, modules_dir, install_dir, make_verbose=False):
+        self.make_verbose = make_verbose
+        super(hostapbuilder, self).__init__(name, modules_dir, modules_dir, install_dir)
+        self.config_src_fpath = os.getcwd() + "/config/" + name + ".config"
```

Comment:
Don't use getcwd; use os.path.abspath:
```suggestion
        self.config_src_fpath = os.path.abspath(os.path.join("config", name + ".config"))
```